.github/workflows/build.yml

```diff
@@ -25,22 +25,25 @@ jobs:
 
   build:
     strategy:
-      # amd64,arm64,ppc64le,s390x
       fail-fast: false
       matrix:
         arch:
           - arch: amd64
             rust-target: x86_64-unknown-linux-gnu
             filename: linux-x86_64
+            command: cargo
```

Comment:
So we don't need cross here since we're building on an x86 system?

Reply:
Correct. That's how I understand it.